### ISSUE TITLE
Gitpod Browser Extension Not Appearing in GitHub

### ISSUE THREAD

**Issue by justin-sumner** [gitpod-io/gitpod#20306]
### Bug description

https://github.com/gitpod-io/browser-extension said issues were being tracked here. Apologies if this is the wrong place.

The Gitpod browser extension doesn't appear to be rendering in GitHub repos. It seems to be working in GitLab though.

Github
![image](https://github.com/user-attachments/assets/d1b413d4-2ee1-4c94-a6c0-e499bdf8ab2b)

Gitlab 
![image](https://github.com/user-attachments/assets/9ee07bfc-aa92-40f1-b0ed-17755fa221df)

### Steps to reproduce

With the Browser Extension enabled navigate to a GitHub repository.

### Expected behavior

The enabled extension adds a Button to open the repository in Gitpod next to the "<> Code" button.

**Comment by jrul222** [comment]
I can also verify the extension isn't working on github.  Have asked multiple people and everyone is getting the same result in both chrome and firefox.

**Comment by christoph-diginav** [comment]
Exact same problem here.

**Comment by VCGithubCode** [comment]
> I can also verify the extension isn't working on github. Have asked multiple people and everyone is getting the same result in both chrome and firefox.

Came to make an identical report

**Comment by mrsimonemms** [comment]
I'm getting the same problem in both Firefox and Brave in GitHub. It does appear to be working on GitLab - I tried "run on all sites" and that didn't do anything.

I'm running on Ubuntu and latest versions of everything.

**Comment by filiptronicek** [comment]
Thanks everyone for the reports, I can reproduce and am working on a fix. It seems that our GitHub friends changed their DOM a bit, so our button injection is having trouble.

**Comment by christoph-diginav** [comment]
Issue is resolved.

**Comment by mrsimonemms** [comment]
Yup, can confirm it's working again. Nice work @filiptronicek - have a little dance

**Comment by filiptronicek** [comment]
@mrsimonemms you bet I will 🕺
